locallibrary/catalog/views.py

```diff
@@ -1 +1,27 @@
 from django.shortcuts import render
+
+from .models import Book, Author, BookInstance, Genre
+
+def index(request):
+    """View function for home page of site."""
+
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
```

Comment:
Loại bỏ bớt method dư thừa nhé em, apply cho các line bên dưới
```suggestion
    num_books = Book.objects.count()
```